style/Makefile

```diff
@@ -2,7 +2,7 @@ clean:
 	rm -rf sprites build icons/us_* rebusurance.zip
 
 rebusurance.zip:
-	wget -q -O rebusurance.zip https://github.com/1ec5/rebusurance/releases/download/v1.0.0/rebusurance-v1.0.0.zip 
+	curl -s -L https://github.com/1ec5/rebusurance/releases/download/v1.0.0/rebusurance-v1.0.0.zip --output rebusurance.zip
 
```

Comment:
This change is fine, but I expect to eventually replace this bulk import with customized-edited shield blanks based on experimentation in another branch.  There's too many changes that need to be made to the artwork to make it look right in 20x20 icons.

Reply:
I just [installed `wget` on my Mac with homebrew](https://formulae.brew.sh/formula/wget), but `curl` works fine too.

Reply:
For some reason my wget can't work with https :( and I thought curl is probably part of any linux distribution anyways...